webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -386,91 +386,91 @@ class Boot extends Loggable {
 
     def policyMenu = {
       val name = "configuration"
-      (Menu(name+"ManagerHome", <i class="fa fa-pencil"></i> ++ <span>{name.capitalize} policy</span>: NodeSeq) /
+      (Menu("20", <i class="fa fa-pencil"></i> ++ <span>{name.capitalize} policy</span>: NodeSeq) /
         "secure" / (name+"Manager") / "index" >> TestAccess ( ()
             => userIsAllowed("/secure/index",AuthorizationType.Configuration.Read) )).submenus (
 
-          Menu(name+"RuleManagement", <span>Rules</span>) /
+          Menu("10", <span>Rules</span>) /
             "secure" / (name+"Manager") / "ruleManagement"
             >> LocGroup(name+"Group")
             >> TestAccess( () => userIsAllowed("/secure/index",AuthorizationType.Rule.Read ) )
 
-        , Menu("Rule Details", <span>Rule</span>) /
+        , Menu("10", <span>Rule</span>) /
```

Comment:
I don't think we should authorize same menu idea, I'm pretty sure it will create unspecified behavior

Reply:
(even for hidden menu)